### PR TITLE
feat: Add views to support EDI

### PR DESCRIPTION
**Pull request by BjarkeMeier**
# Description

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [x] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_edi_energy_results_per_brp_ga_v1.sql
Comment (johevemi) [inline source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_edi_energy_results_per_brp_ga_v1.sql:1]: Should there no be feature test on views?

File: source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_edi_energy_results_per_es_brp_ga_v1.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+       metering_point_type,
+       settlement_method,
+       resolution,
+       time,
+       quantity,
+       unit,
+       quantity_qualities
+FROM {OUTPUT_DATABASE_NAME}.succeeded_energy_results_v1
+WHERE
+    calculation_type in ('BalanceFixing', 'Aggregation')
```
Comment (JesperGomes) [inline source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_edi_energy_results_per_es_brp_ga_v1.sql:20]: Why aren't all calc types included? I thought energy suppliers also needed this for wholesale calculations?

Comment (BjarkeMeier) [reply source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_edi_energy_results_per_es_brp_ga_v1.sql:20]: Nice catch!

Comment (BjarkeMeier) [reply source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_edi_energy_results_per_es_brp_ga_v1.sql:20]: @khatozen, see this. I'll update the contract (adding all calculation types) in Confluence and this PR.

Comment (khatozen) [reply source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_edi_energy_results_per_es_brp_ga_v1.sql:20]: @BjarkeMeier: That is fine with me, if we are still using **es_brp_ga** for energy resultes for energy supplier in wholesale calculation.

File: source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_edi_energy_results_per_brp_ga_v1.sql
Comment (BjarkeMeier) [reply source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_edi_energy_results_per_brp_ga_v1.sql:1]: You're right. I'll add them to the PR.

**Review by johevemi — APPROVED** [review]

**Pull request merged**